lib/beaker/host/pswindows.rb

```diff
@@ -28,46 +28,46 @@ def self.pe_defaults
         })
 
-        if platform.include?('amd64')
-          h.merge({
-            'puppetbindir'  => 'C:\\Program Files (x86)\\PuppetLabs\\Puppet Enterprise\\bin'
-            })
-          else
-            h.merge({
-              'puppetbindir'  => 'C:\\Program Files\\PuppetLabs\\Puppet Enterprise\\bin'
-              })
-            end
-          end
+      if platform.include?('amd64')
+        h.merge({
+          'puppetbindir'  => 'C:\\Program Files (x86)\\PuppetLabs\\Puppet Enterprise\\bin'
+          })
+      else
+        h.merge({
+          'puppetbindir'  => 'C:\\Program Files\\PuppetLabs\\Puppet Enterprise\\bin'
+          })
+      end
+    end
 
-          def self.foss_defaults
-            h = Beaker::Options::OptionsHash.new
-            h.merge({
-              'user'              => 'Administrator',
-              'group'             => 'Administrators',
-              'distmoduledir'     => 'C:\\ProgramData\\PuppetLabs\\puppet\\etc\\modules',
-              'sitemoduledir'     => 'C:\\usr\\share\\puppet\\modules',
-              'hieralibdir'       => 'C:\\opt\\puppet-git-repos\\hiera\\lib',
-              'hierapuppetlibdir' => 'C:\\opt\\puppet-git-repos\\hiera-puppet\\lib',
-              'hierabindir'       => 'C:\\opt\\puppet-git-repos\\hiera\\bin',
-              'pathseparator'     => ';'
-              })
+    def self.foss_defaults
+      h = Beaker::Options::OptionsHash.new
+      h.merge({
+        'user'              => 'Administrator',
+        'group'             => 'Administrators',
+        'distmoduledir'     => 'C:\\ProgramData\\PuppetLabs\\puppet\\etc\\modules',
+        'sitemoduledir'     => 'C:\\usr\\share\\puppet\\modules',
+        'hieralibdir'       => 'C:\\opt\\puppet-git-repos\\hiera\\lib',
+        'hierapuppetlibdir' => 'C:\\opt\\puppet-git-repos\\hiera-puppet\\lib',
+        'hierabindir'       => 'C:\\opt\\puppet-git-repos\\hiera\\bin',
+        'pathseparator'     => ';'
+        })
 
-              if h['platform'] && h['platform'].include?('amd64')
-                h.merge({
-                  'puppetpath'    => "C:\\Program Files (x86)\\Puppet Labs\\Puppet\\etc",
-                  'hieraconf'     => "C:\\Program Files (x86)\\Puppet Labs\\Puppet\\etc\\hiera.yaml",
-                  'puppetvardir'  => 'C:\\Program Files (x86)\\Puppet Labs\\Puppet\\var',
-                  'puppetbindir'  => "C:\\Program Files (x86)\\Puppet Labs\\Puppet\\bin"
-                  })
-                else
-                  h.merge({
-                    'puppetpath'    => "C:\\Program Files\\Puppet Labs\\Puppet\\etc",
-                    'hieraconf'     => "C:\\Program Files\\Puppet Labs\\Puppet\\etc\\hiera.yaml",
-                    'puppetvardir'  => 'C:\\Program Files\\Puppet Labs\\Puppet\\var',
-                    'puppetbindir'  => "C:\\Program Files\\Puppet Labs\\Puppet\\bin"
-                    })
-                  end
+      if h['platform'] && h['platform'].include?('amd64')
```

Comment:
I hit this myself, but I don't think that this is the right fix.  Here you are checking h for a platform setting, but you just created h - it will have no platform setting and you will never hit this chunk of code.